### ISSUE TITLE
TypeError:  WebPushException does not take keyword arguments

### ISSUE THREAD

**Issue by marauder37** [safwanrahman/django-webpush#56]
You can't pass extra= anything when raising WebPushException.

In django-webpush utils.py:

```
utils.py:22:raise WebPushException("Push failed.", extra=errors)
utils.py:40:raise WebPushException("Push failed.", extra=errors)
```

but in pywebpush __init__.py:

```
class WebPushException(Exception):
    pass
```

**Comment by marauder37** [comment]
Also, this is the traceback for the causing exception. 

Have you thought about the best way to handle these 410 responses?

![pusherr](https://user-images.githubusercontent.com/1592348/53053729-4a05bd00-34f6-11e9-86c0-f3d750708cc3.png)

**Comment by jonderwaater** [comment]
I have the same error. Strangely the push notifications is sent, even though this exception is raised.

**Comment by marauder37** [comment]
@jonderwaater unfortunately that will not be reliable. You may/may not receive each notification. For our users it looks random whether the notification is received.

One root cause of the exception is having a webpush subscription in Django that is no longer registered in the browser. Notifications are sent to each subscription in sequence. So if you have two subscriptions, one revoked and one active, sometimes the notification will be sent to the active subscription before the exception is raised. But if the revoked subscription is first in the queryset, the exception will be raised before the notification is sent, and you won't get it.

**Comment by safwanrahman** [comment]
Thanks for the issue. I am working on a solution that will delete or update the revoked webpush permission.

**Comment by jonderwaater** [comment]
I implemented a solution here: https://github.com/jonderwaater/django-webpush/tree/20190329-handle_410_response
Probably broke some other things along the way though.

**Comment by safwanrahman** [comment]
Sorry for the issue. It has been fixed with  09996c0
